platform/feature-api/communication/postoffice/src/types.ts

```diff
@@ -1,52 +1,52 @@
 /**
- * This module defines the types used for specifying endpoints.
+ * This module defines the types used for specifying backend endpoints.
  *
- * An _endpoint_ is an API that can be implemented by a server and called by a
- * client. Endpoints are structured as objects with methods; methods may return
+ * A _backend endpoint_ is an API that can be implemented by a server and called by a
+ * client. Backend endpoints are structured as objects with methods; methods may return
  * either values or a nested endpoint.
  *
  * An _endpoint specification_ is a mere type that is never used directly. See
- * [[EndpointSpec]] for details.
+ * [[BackendSpec]] for details.
  *
  * @packageDocumentation
  */
 
 /**
- * Interface denoting a value endpoint of type `T`. This interface is purely
+ * Interface denoting a value backend endpoint of type `T`. This interface is purely
  * virtual and no instances are generated.
  */
-export interface ValueEndpointSpec<T> {
+export interface ValueBackendSpec<T> {
     endpointType: "value";
     value: T;
 }
 
 /**
- * Interface denoting a method endpoint of type `T`. This interface is purely
+ * Interface denoting a method backend endpoint of type `T`. This interface is purely
  * virtual and no instances are generated.
  */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export interface ObjectEndpointSpec<T extends Record<string, (...args: any[]) => EndpointSpec>> {
+export interface ObjectBackendSpec<T extends Record<string, (...args: any[]) => BackendSpec>> {
     endpointType: "object";
     methods: T;
 }
 
 /**
- * An endpoint specification describes either a value ([[ValueEndpointSpec]]) or
- * an object ([[ObjectEndpointSpec]]). Values have no further structure and are
+ * A backend endpoint specification describes either a value ([[ValueBackendSpec]]) or
+ * an object ([[ObjectBackendSpec]]). Values have no further structure and are
  * considered to be a return value representing the final response to the
  * client. Objects have methods that can be called by a client.
  *
- * An endpoint call is structured as `endpoint.f(...).g(...)`; i.e. the endpoint
+ * A backend endpoint call is structured as `endpoint.f(...).g(...)`; i.e. the backend endpoint
  * object followed by a non-empty sequence of method calls with parameters. The
  * result of this chain is the _value_ that is returned to the client.
  *
  * Example:
  *
  * ```
- * type SimpleEndpoint = ObjectEndpointSpec<{
- *     test1(param1: string): ValueEndpointSpec<number>;
- *     test2(param1: string): ValueEndpointSpec<number>;
- *     test3(parama: boolean, ...paramb: number[]): ValueEndpointSpec<string>;
+ * type SimpleEndpoint = ObjectBackendSpec<{
+ *     test1(param1: string): ValueBackendSpec<number>;
```

Comment:
If I understood the definition of ObjectBackEndSpec correctly, this is not correct (before and now). It should be something like:

```
type SimpleEndpoint = ObjectEndpointSpec<{
     test1: test1(param1: string): ValueEndpointSpec<number>,
     test2: test2(param1: string): ValueEndpointSpec<number>,
     test3: test3(parama: boolean, ...paramb: number[]): ValueEndpointSpec<string>
}
```

An `ObjectEndPointSpec` is a `Record`, right?